test/unit/store/cart-store.js

```diff
@@ -0,0 +1,31 @@
+import { Dispatcher } from 'consus-flux';
+import CartStore from '../../../.dist/store/cart-store';
+import { assert } from 'chai';
+
+describe('CartStore', () => {
+
+    it('should instantiate without any items', () => {
+        assert.strictEqual(CartStore.getItems().length, 0);
+    });
+
+    it('should add item to list', () => {
+        Dispatcher.handleAction('CHECKOUT_ITEM_FOUND',{
```

Comment:
On second thought, this makes sense too. The action is that we found an item for checkout. It should be up to the store to actually validate the item (e.g. making sure the item isn't already in the cart) and then add it to the cart.

:+1: